services/device-quota-suggestion-service/app/service.py

```
@@ -0,0 +1,205 @@
+        embeddings = self.embedding_backend.embed(normalized)
+        vectors = [
+            CategoryVector(
+                category=category,
+                normalized_name=normalized_name,
+                embedding=embedding,
+            )
+            for category, normalized_name, embedding in zip(
+                request.categories,
+                normalized,
+                embeddings,
+            )
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)